### DISCUSSION TITLE
governed table row level update

### DISCUSSION THREAD

**Discussion by alwem82** [aws/aws-sdk-pandas#1244] [Q&A] [answered]
Hello,

I am aware that function for "governed table row level update" is in preview stage.
Is there any way we could test row level update function with wrangler API?

preview permission is granted.

**Comment by kukushking** [comment, marked as answer]
Hi @alwem82 no unfortunately at the moment only [these](https://aws-data-wrangler.readthedocs.io/en/2.15.0/api.html#aws-lake-formation) functions are available
